ui/app/styles/components/page-layout.scss

```diff
@@ -36,7 +36,6 @@
 
       &.is-right {
         margin-left: $gutter-width;
-        overflow-x: auto;
```

Comment:
Did you mean to include this here? Would this mean you can't pan on mobile anymore?

Reply:
I did mean to include this. It was causing the icon tooltip to be clipped on the left side where it goes out of the bounds of the page-column.

As for panning on mobile, good question. I just checked and it appears to still work.